server/src/main/scala/manifests/passengers/BestManifests.scala

```diff
@@ -20,14 +20,28 @@ case class BestAvailableManifest(source: String,
                                  passengerList: List[ManifestPassengerProfile])
 
 object BestAvailableManifest {
-  def apply(manifest: VoyageManifest): BestAvailableManifest = BestAvailableManifest(
-    SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages,
-    manifest.ArrivalPortCode,
-    manifest.DeparturePortCode,
-    manifest.VoyageNumber,
-    manifest.CarrierCode,
-    manifest.scheduleArrivalDateTime.getOrElse(SDate.now()),
-    manifest.PassengerList.map(p => ManifestPassengerProfile(p, manifest.ArrivalPortCode)))
+  def apply(manifest: VoyageManifest): BestAvailableManifest = {
```

Comment:
I think there's a case missing here. We sometimes get manifests containing passengers with and without the `PassengerIdentifier` specified. When this is the case we only care about the ones with an identifier and so want to filter out the ones that don't have one. So it's not so much that we get duplicate identifiers - it's that we get the same passenger both with an identifier and without one, ie iAPI & non-iAPI

Reply:
nice catch - I've added this case.